src/backend/elfobj.c

```diff
@@ -3495,6 +3493,8 @@ void Obj::gotref(symbol *s)
     }
 }
 
+#endif
```

Comment:
Did you unintentionally revert https://github.com/WalterBright/dmd/commit/2c04a422e0ec2d5a291ce29fc19cff8706941c01 ?